bouture.js

```diff
@@ -1,61 +1,98 @@
 
-tagNames.forEach(tag => {
-  Object.defineProperty(Bouture, tag, {
-    get: () => {
-      const element = document.createElement(tag)
+tagNames.forEach(tagName => {
+  Object.defineProperty(Bouture, tagName, {
+    get: function () {
+      let tags = [{name: tagName, args: []}]
       function branche (...args) {
-        args.forEach(arg => {
-          switch (typeof arg) {
-            case 'string':
-              element.append(arg)
-              break
-            case 'number':
-              if (!Number.isNaN(arg)) element.append(arg)
-              break
-            case 'object':
-              if (arg !== null) {
-                Object.keys(arg)
-                  .forEach(attributeName => {
-                    const attributeValue = arg[attributeName]
-                    switch (typeof attributeValue) {
-                      case 'boolean':
-                        if (attributeValue) {
-                          element.setAttribute(attributeName, '')
-                        }
-                        break
-                      case 'number':
-                        if (!Number.isNaN(attributeValue)) {
-                          element.setAttribute(attributeName, attributeValue)
-                        }
-                        break
-                      case 'string':
-                        element.setAttribute(attributeName, attributeValue)
-                        break
-                      case 'object':
-                        if (attributeValue !== null) {
-                          element
-                            .setAttribute(attributeName, attributeValue.join(' '))
-                        }
-                        break
-                      case 'symbol':
-                        break
-                      case 'undefined':
-                        break
-                    }
-                  })
-              }
-              break
-            default:
-              // Other types not handle by cases : Symbol, boolean, undefined
-          }
-        })
+        const lastTag = tags.pop()
+        tags.push({name: lastTag.name, args: args})
```

Comment:
The `pop` + `push` feels a bit wasteful (removing an element and adding it back in)
What about mutating the last element?